### PR TITLE
Ersilia Catalog Bug when no models are available

### PR DESCRIPTION
**Pull request by dzumii**
**Description**
When you try to run ersilia catalog --local but no models are available locally, instead of telling you there is no local model available, it outputs an error:

**Changes Made**
-updated the `local` method to return an empty `catalogue_table` if no models are found.
-updated the `catalogue` command to print a message when the `catalogue_table` has no data

**Status**
"No local model is available. Please fetch a model by running 'ersilia fetch ...'" is returned when you run  `ersilia catalog --local` and no local model is available 

Related to #1072

### CODE REVIEW

File: ersilia/cli/commands/catalog.py [hunk 1]
```diff
@@ -43,15 +45,25 @@ def catalog(local=False, file_name=None, browser=False, more=False):
             mc.airtable()
             return
         if local:
+            catalog_table = mc.local()
```
Comment (DhanshreeA) [inline ersilia/cli/commands/catalog.py:48]: I like your use of storing `mc.local` and `mc.hub` into a `catalog_table` variable. Could we maybe rewrite this whole bit as:

```Python

catalog_table = mc.local() if local else mc.hub()

if file_name is None:
    catalog = catalog_table.as_json()
else:
    catalog_table.write(file_name)
    catalog = None
```

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

File: ersilia/cli/commands/catalog.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import click
 
 from . import ersilia_cli
+from .. import echo
```
[the remark this answers is not in the document]
Comment (dzumii) [reply ersilia/cli/commands/catalog.py:4]: Oh! I was only trying something out,forgot to remove it

**Review by DhanshreeA — APPROVED** [review]

**Comment by DhanshreeA** [comment]
LGTM!

File: ersilia/cli/commands/catalog.py (hunk 1 above)
Comment (dzumii) [reply ersilia/cli/commands/catalog.py]: Alright

**Pull request merged**